### PR TITLE
[SPARK-28802][DOC][SQL]Document DESCRIBE DATABASE statement in SQL Reference

### PR DESCRIPTION
**Pull request by kevinyu98**
### What changes were proposed in this pull request?
Document DESCRIBE DATABASE statement in SQL Reference

### Why are the changes needed?

To complete the SQL Reference

### Does this PR introduce any user-facing change?
Yes

#### Before
There is no documentation for this command in sql reference

#### After
Screen Shot 2019-09-05 at 12 59 32 PM
Screen Shot 2019-09-05 at 12 59 45 PM

### How was this patch tested?
Used jekyll build and serve to verify

### CODE REVIEW

**Comment by dilipbiswal** [comment]
@kevinyu98 Can you please attach a screen-shot of the page ?

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 1]
```diff
@@ -18,5 +18,23 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
+​
+Return the metadata of an existing database(name, comment and location). If the database does not exist,
```
Comment (srowen) [inline docs/sql-ref-syntax-aux-describe-database.md]: Space after database

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 2]
```diff
@@ -18,5 +18,23 @@ license: |
...
+an exception is thrown.
+
+When `extended` is specified, it also shows the database's properties.
```
Comment (srowen) [inline docs/sql-ref-syntax-aux-describe-database.md]: `EXTENDED` for consistency

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 3]
```diff
@@ -18,5 +18,23 @@ license: |
...
+You can use the abbreviation `DESC` for the `DESCRIBE` statement.
+
+### Syntax
+{% highlight sql %}
+[DESC | DESCRIBE] DATABASE [EXTENDED] db_name 
+{% endhighlight %}
+
+### Example
+{% highlight sql %}
+DESCRIBE DATABASE EXTENDED tempdb
```
Comment (srowen) [inline docs/sql-ref-syntax-aux-describe-database.md]: Can you provide any output? this doc page isn't giving much additional info (though there is not a lot to know, granted)

**Review by srowen — CHANGES_REQUESTED** [review]

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 1 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: ok.

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 2 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: sure

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 3 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: sure

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 4]
```diff
@@ -18,5 +18,79 @@ license: |
...
+  <dd>
+    Specifies a name of an existing database or an existing schema in the syetem. If the name does not exist, an
+    exception is thrown. The name is case insensitive, it is stored as low case in SPARK<br><br>
+  </dd>
+</dl>
+
+### Example
+{% highlight sql %}
+-- Create employees DATABASE
+CREATE DATABASE EMPLOYEES COMMENT 'For software companies';
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-describe-database.md]: @kevinyu98 lets use lowercase for `EMPLOYEES` to match other PRS..

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 5]
```diff
@@ -18,5 +18,79 @@ license: |
...
+
+-- Describe employees DATABASE.
+-- Returns Database Name, Description and Root location of the filesystem for the employees DATABASE
+DESCRIBE DATABASE Employees;
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-describe-database.md]: @kevinyu98 ditto.

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 6]
```diff
@@ -18,5 +18,79 @@ license: |
...
+  </dd>
+</dl>
+
+### Example
+{% highlight sql %}
+-- Create employees DATABASE
+CREATE DATABASE EMPLOYEES COMMENT 'For software companies';
+
+-- Describe employees DATABASE.
+-- Returns Database Name, Description and Root location of the filesystem for the employees DATABASE
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-describe-database.md]: @kevinyu98 Can we please put DATABASE in another line with a comment marker. ?

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 7]
```diff
@@ -18,5 +18,79 @@ license: |
...
+DESCRIBE DATABASE Employees;
++-------------------------+-----------------------------+
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-describe-database.md]: @kevinyu98 Can we keep the results 2 spaces to the right to match other PRs.

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 8]
```diff
@@ -18,5 +18,79 @@ license: |
...
+|Database Name            |employees                    |
+|Description              |For software companies       |
+|Location                 |file:/Users/Temp/employees.db|
++-------------------------+-----------------------------+
+{% endhighlight %}
+
+{% highlight sql %}
+-- Create employees DATABASE
+CREATE DATABASE EMPLOYEES COMMENT 'For software companies';
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-describe-database.md]: @kevinyu98 lowercase employees.

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 9]
```diff
@@ -18,5 +18,79 @@ license: |
...
+
+-- Alter employees database to set DBPROPERTIES
+ALTER DATABASE EMPLOYEES SET DBPROPERTIES ('Create-by' = 'Kevin', 'Create-date' = '09/01/2019');
+
+-- Describe employees DATABASE with EXTENDED option to the database properties
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-describe-database.md]: @kevinyu98 "to return additional database properties"

File: docs/sql-ref-syntax-aux-describe-database.md [hunk 10]
```diff
@@ -18,5 +18,79 @@ license: |
...
+|Database Name            |employees                                    |
+|Description              |For software companies                       |
+|Location                 |file:/Users/Temp/employees.db                |
+|Properties               |((Create-by,kevin), (Create-date,09/01/2019))|
++-------------------------+---------------------------------------------+
+{% endhighlight %}
+
+{% highlight sql %}
+-- Create deployment SCHEMA
+CREATE SCHEMA DEPLOYMENT comment 'Deployment environment';
```
Comment (dilipbiswal) [inline docs/sql-ref-syntax-aux-describe-database.md]: @kevinyu98 lowercase ..

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 4 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: @dilipbiswal sure

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 6 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: ok

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 5 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: changed

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 7 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: changed

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 8 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: ok

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 9 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: ok

File: docs/sql-ref-syntax-aux-describe-database.md (hunk 10 above)
Comment (kevinyu98) [reply docs/sql-ref-syntax-aux-describe-database.md]: ok

**Comment by dilipbiswal** [comment]
LGTM
cc @gatorsmile

**Comment by gatorsmile** [comment]
ok to test

**Comment by gatorsmile** [comment]
LGTM

Thanks! Merged to master.

**Comment by kevinyu98** [comment]
@gatorsmile thank a lot.

**Comment by dilipbiswal** [comment]
retest this please

**Pull request closed without merging**